### PR TITLE
perlgov: the perl governance document

### PR DESCRIPTION
**Pull request by rjbs**
This is the perlgov document approved by the perl governance bootup list!

https://perl.topicbox.com/groups/gov-begin/T9df40cacd314f378/sixth-ballot-has-been-sent

### CODE REVIEW

**Comment by Grinnz** [comment]
I don't know if this is subject to modification since it's been voted on already but it should probably have a standard DESCRIPTION section.

File: pod/perl.pod
Comment (Grinnz) [inline pod/perl.pod]: This seems like an odd place (internals) but I guess it was placed next to perlpolicy? Either way the indentation is off.

File: pod/perlgov.pod
Comment (Grinnz) [inline pod/perlgov.pod]: This must be removed from the NAME top level section.

**Comment by philiprbrenan** [comment]
Please:

   s/Perl Language and Interpreter/Perl Language, Interpreter and CPAN/g 

because without CPAN Perl is not very useful.

**Comment by philiprbrenan** [comment]
Please make votes public so we can have confidence in them. 

s/ Voting is performed anonymously./Voting is performed publicly./g

**Comment by philiprbrenan** [comment]
Tony Benn -  Five questions to ask those in power:

    What power have you got?
    Where did you get it from?
    In whose interests do you exercise it?
    To whom are you accountable?
    And how can we get rid of you?

Please answer these questions convincingly, especially the last one, if you wish your new organization to enjoy widespread support?

**Comment by Grinnz** [comment]
> Please:
> 
> s/Perl Language and Interpreter/Perl Language, Interpreter and CPAN/g
> 
> because without CPAN Perl is not very useful.

CPAN is out of the scope of this document regardless of its usefulness.

File: pod/perlgov.pod
Comment (ericherman) [inline pod/perlgov.pod:277]: This seems plenty good enough as-is. I believe the spirit of this would be better phrased as "Inactive
members may declare themselves active at any time, but will not participate in votes which began while they were inactive." This or similar wording would also allow re-activation even during periods of concurrent or long-running votes.

**Review by ericherman — APPROVED** [review]

**Comment by wbraswell** [comment]
It doesn't matter how well-written your governance document is, or how many publicly-archived mailing lists you set up, or how many "votes" your group gives itself.

This is nothing more than a shiny new mask to be worn by the same old self-appointed cabal.

For starters, we have over 4.7K Perl programmers on Facebook, and never once was our input solicited.

Here are a few quotes from prominent members of the Perl community:

`"I'm disappointed that the community continues to have so little say. The inner circle that appoints itself continues."`

`"There can only be one lord of the ring and he does not share power."`

`"Whatever. Can we get back to Perl 7 and Cor, please?"`

https://www.facebook.com/groups/perlprogrammers/permalink/3781573088542118

**Comment by wbraswell** [comment]
@philiprbrenan makes some extremely pertinent points, so I'll go ahead and venture a guess at the following answers:

`"Please make votes public so we can have confidence in them."`

Sorry Philip, but the Perl Governance Cabal (PGC) does not seem to be interested in allowing you to know their actual opinions or votes on important issues.  This is also the tried-and-true method of keeping everyone in the dark, as currently employed by TPF.

**Comment by wbraswell** [comment]
`"What power have you got?"`

The Perl Governance Cabal (PGC) has absolute power over all technical decisions related to the Perl interpreter software, and by extension, the future direction of the entire Perl language, community, and ecosystem.

`"Where did you get it from?"`

The PGC, for many years, has been a self-appointed government.  They derive their power from their own mandate.

`"In whose interests do you exercise it?"`

In general, self-appointed governments such as PGC will exercise their powers in their own interest.

`"To whom are you accountable?"`

The PGC does not appear to be accountable to anyone in any way.

`"And how can we get rid of you?"`

There does not appear to be any option for removing the PGC from power, so we are all out of luck.  If you don't like PGC, then perhaps Perl is not the language for you?

**Comment by karenetheridge** [comment]
Please, everyone, keep it civil. Discussions of the governance model itself that do not relate to the technical details of this pull request do not belong in this ticket. Please feel free to start a thread on the p5p mailing list, or any other venue you like (facebook, reddit, blogs.perl.org etc).

**Comment by wbraswell** [comment]
@karenetheridge 

I have 3 replies to your suggestion:

1.  Your name is on the PGC membership list.  This makes it very hard to believe that you are, in any way, neutral or unprejudiced in these matters.  It is safe to assume that, as a PGC member, you will act in ways which are in the best interest of the PGC.

2.  Clearly, creating discussion threads anywhere other than this official GitHub repository will have little to no effect, and will be completely ignored by the PGC.  But sure, we're actively discussing this on Facebook, the link for which I have already provided.

3.  Which comments, exactly, do you find to not be "civil", and why?

**Comment by karenetheridge** [comment]
> Your name is on the PGC membership list.

I'm on the "core team" because I contribute to Perl core. That's it.

> Clearly, creating discussion threads anywhere other than this official GitHub repository will have little to no effect

Why would I make the suggestion if I was planning on ignoring the feedback?

> Which comments, exactly, do you find to not be "civil", and why?

I just quoted one. Using insulting words such as "cabal" is another, and making allegations that you have not backed up with evidence.

I'm locking this thread. Anyone who has any comments about the PR specifically is still welcome to make them.

File: pod/perl.pod
Comment (rjbs) [reply pod/perl.pod]: Correct, next to perlpolicy.  The indentation was my bad tabstop setting. :)

File: pod/perlgov.pod
Comment (rjbs) [reply pod/perlgov.pod]: Done.

File: pod/perlgov.pod
Comment (rjbs) [reply pod/perlgov.pod:277]: I take your point, but believe this will be exceptionally rare in practice.  If it does come up, I will propose a change such as this.  Thanks.

**Comment by rjbs** [comment]
@xsawyerx I believe this remains the correct patch, but would not mind your :+1: here!

**Comment by lightsey** [comment]
Aside from technical issues with formatting, this pull request is not a proper venue to address disagreements over the intent or structure of the governance model defined by this document.

The process for changing the governance rules is defined in the governance rules. Even if you make valid arguments for changing the governance rules in this pull request, it would be inappropriate for @rjbs to alter the governance rules in response to them.

**Comment by rjbs** [comment]
@xsawyerx @neilb I would like to merge this in the next 1-3d unless there is objection.

**Comment by xsawyerx** [comment]
+1 from me. No major comments that weren't discussed already in the past.

**Comment by rjbs** [comment]
Rebased, grindperl-ing, will merge if all goes well.

**Pull request merged**
